packages/muelu/example/basic/CMakeLists.txt

```diff
@@ -5,7 +5,7 @@ INCLUDE_DIRECTORIES(${CMAKE_CURRENT_SOURCE_DIR})
 #
 
 IF(HAVE_MUELU_INST_DOUBLE_INT_INT)
```

Comment:
Y'all realize that this test will never get enabled in the future, right?

Reply:
@mhoemmen we won't allow `int_int` only `long_int_int` and `long_long_int_int` ?

Reply:
That's correct, but this test only gets enabled for `int_int` right?